### PR TITLE
Created header with menu

### PR DESCRIPTION
**Pull request by Zelinskyi-Serhii**
https://Zelinskyi-Serhii.github.io/layout_miami/

### CODE REVIEW

**Comment by Yevhenii0536** [comment]
image
your bg-color doesn't as in mock-up

**Comment by Yevhenii0536** [comment]
image
this is must be as link, wich will bring users to page with another product with same category

**Comment by Yevhenii0536** [comment]
image
in mock-up title of products must have left position. in yours - title have position center

**Review by Yevhenii0536 — CHANGES_REQUESTED** [review]
pls, fix issues which i was write

**Review by Yevhenii0536 — APPROVED** [review]

File: src/blocks/button.scss [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      color: #fff;
+    }
+
+    @include onTablet {
+      width: 392px;
+    }
+
+    @include onTablet {
+      width: 324px;
+    }
```
Comment (Andr1yK) [inline src/blocks/button.scss:42]: looks weird)

**Review by Andr1yK — CHANGES_REQUESTED** [review]
bug on submit form

https://user-images.githubusercontent.com/86183007/218504802-4a922e94-7ccd-42bd-9942-3dfc23f18ba2.mov

add hovers

image

**Review by Andr1yK — CHANGES_REQUESTED** [review]
fix links and form (points 13-15 in the checklist)

https://user-images.githubusercontent.com/86183007/219872944-34f42048-9f40-4a05-a074-f9eb6d8b378b.mov

File: src/blocks/header.scss [hunk 2]
```diff
@@ -0,0 +1,109 @@
+.header {
+  background-image: url(../images/headerMobileSvg.svg);
```
Comment (sergii-nosachenko) [inline src/blocks/header.scss:2]: Why do you use vector format for the raster image?

**Review by sergii-nosachenko — CHANGES_REQUESTED** [review]
Some issues:
- check raster image format (for photos JPEG is preferable)
- check the images ration after the tablet breakpoint
   
Bose (1)

- check menu z-index

https://www.loom.com/share/8b1741f4708140faa1827d7c241f2905

**Review by sergii-nosachenko — APPROVED** [review]
Good work 👍

**Pull request closed without merging**